### ISSUE TITLE
GitHub roadmap issue with title but no linked milestone issues should still render in starmaps

### ISSUE THREAD

**Issue by juliaxbow** [pln-planning-tools/Starmaps#104]
**Context**
This error came up during testing. User had created a placeholder "Roadmap" root node issue that had no child milestone issues linked. Description read "tbd"

**What happened**
User received a "Not Found" error message per
`    if (_.isArray(children) && children.length === 0) reject('empty array');`

**What should happen**
Presuming other requirements are met for this parent node (i.e. the starmaps label is attached), on searching starmaps with an issue where no child milestone issues exist:
- the title should render in starmaps
- a link to the github issue should render
- an empty timeline should appear in starmaps

tl;dr: the tool should function as normal but with no child milestones populating.

**Comment by SgtPooki** [comment]
worth discussion at a later point, but for now, let's require ETA and the "starmaps" label

**Comment by SgtPooki** [comment]
I believe this one should be resolved.
